apps/wallet-dashboard/components/dialogs/staking/views/EnterTimelockedAmountView.tsx

```diff
@@ -68,9 +76,44 @@ export function EnterTimelockedAmountView({
         CoinFormat.FULL,
     );
 
+    const [possibleAmountFormatted, possibleAmountSymbol] = useFormatCoin(
+        possibleAmount,
+        IOTA_TYPE_ARG,
+        CoinFormat.FULL,
+    );
+
     const caption = `${maxTokenFormatted} ${maxTokenFormattedSymbol} Available`;
-    const infoMessage =
-        'It is not possible to combine timelocked objects to stake the entered amount. Please try a different amount.';
+    const info = useMemo(() => {
+        if (isSearchingProtocolMaxAmount) {
+            let message = 'Due to the large number of objects current amount is not valid. ';
+
+            message += isTransactionLoading
+                ? 'We are determining a valid amount.'
+                : `A valid amount: ${possibleAmountFormatted} ${possibleAmountSymbol}`;
+
+            return {
+                title: 'Partial staking',
+                message: message,
+            };
+        }
+
+        if (!hasGroupedTimelockObjects) {
+            return {
+                message:
+                    'It is not possible to combine timelocked objects to stake the entered amount. Please try a different amount.',
```

Comment:
This line was not changed(I think that I added it 😄 ) but we could update it to : `Combining timelocked objects to stake the entered amount is not possible. Please try a different amount.`